### PR TITLE
fix on permissions submission so user gets warning of dangerous perms

### PR DESCRIPTION
**Pull request by herbdool**
Fixes Issue #23

### CODE REVIEW

**Comment by herbdool** [comment]
@jackaponte if you've got a chance can you test this?

**Comment by herbdool** [comment]
@colemanw this fix is trivial and should be good to merge.

File: civicrm.module [hunk 1]
```diff
@@ -1029,13 +1029,13 @@ function civicrm_form_alter(&$form, &$form_state, $formID) {
  * @param $form_state
  */
 function civicrm_user_admin_permissions_submit($form, &$form_state) {
-  $rid = array_search('anonymous user', $form_state['values']['role_names']);
-  if ($rid === FALSE) {
+  $role = $form_state['values']['roles']['anonymous'];
+  if ($role === FALSE) {
```
Comment (colemanw) [inline civicrm.module]: This doesn't look like a safe change to me. This condition **was** testing whether the results of `array_search` were `FALSE`. Not sure what it's testing now, but doesn't look right.

**Review by colemanw — DISMISSED** [review]

**Comment by herbdool** [comment]
@colemanw actually that if statement isn't needed now that I look at it more closely. The submit handler only checks anonymous permissions.

I should add that the anonymous class always exists in this function as far as I can determine so there should be no need to exit early.

**Pull request merged**